async-query-core/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what is this for? and nit: add newline in the end

Reply:
This is a magic spell to enable mocking on final class with Mockito. (Mockito disable it by default, and can be enabled with this configuration file. And we are doing the same for other modules.)